src/utils/events/move-input.js

```diff
@@ -24,9 +26,29 @@ export default class MoveInput {
   }
 
   handler(event) {
```

Comment:
The name is from hammer.js, which uses it as an abstract method on its input classes. So, while it's not idiomatic in our frameworks, it is so for Hammer...